CHANGES.md

```diff
@@ -59,6 +59,9 @@
   diff` command can be used to just display the promotion without applying it.
   (#6160, fixes #5368, @emillon)
 
+- Coq native mode is now automatically detected by Dune starting with Coq lang
+  0.7 (#6409, @Alizter)
```

Comment:
maybe add ", thus, the `(mode native)` field has been deprecated and the automatic disabling of native compilation in `dev` is removed in favor of the Coq configure switch"